_co-op-companies/N-Ubisoft.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Ubisoft"
+identification: "ubisoft"
+description: "Ubisoft is a entertainment software company."
+link: "https://winnipeg.ubisoft.com/"
+image: "assets/img/logos/Ubisoft.png"
+width: "200px"
+complete: true
+members:
+  -name: "Raquel"
```

Comment:
Can you add Raquel's last name there too? It's **Theissen**. Thanks!

Reply:
Added!